.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: "* */12 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: -1
+          operations-per-run: 300
```

Comment:
Is this a okay amount? :eyes:

Reply:
I dunno, gatsby is at 5000. The default is 30.

Reply:
Ah well if gatsby is at 5000 then 300 is fine :+1: